src/dialects/mssql/query-generator-typescript.ts

```diff
@@ -83,4 +83,69 @@ export class MsSqlQueryGeneratorTypeScript extends AbstractQueryGenerator {
       this.quoteTable(tableName),
     ]);
   }
+
+  /**
+   * Generates an SQL query that returns all foreign keys of a table.
+   *
+   * @param   tableName   The table or associated model.
+   * @returns             The generated SQL query.
+   */
+  getForeignKeysQuery(tableName: TableNameOrModel) {
+    const table = this.extractTableDetails(tableName);
+    // TODO: research if it's possible to get the database from the provided tableName
```

Comment:
not yet! But it's something we need to do for both mssql & postgres. See https://github.com/sequelize/sequelize/issues/12449#issuecomment-1019327167

Reply:
I've updated the TODO to reference that issue